back/app/services/input_ui_schema_generator_service.rb

```diff
@@ -14,7 +14,9 @@ def visit_html_multiloc(field)
   protected
 
   def generate_for_current_locale(fields)
-    input_term = fields.first.resource.participation_context&.input_term || ParticipationContext::DEFAULT_INPUT_TERM
+    participation_context = fields.first.resource.participation_context
```

Comment:
The removed line implies that the participation context can be nil. In that case I don't think the participation method should be instantiated.